mmv1/products/compute/NetworkFirewallPolicyRule.yaml

```diff
@@ -0,0 +1,295 @@
+          type: String
+      - name: 'srcFqdns'
+        type: Array
+        immutable: true
+        description: |
+          Fully Qualified Domain Name (FQDN) which should be matched against traffic source. Maximum number of source fqdn allowed is 100.
+        item_type:
+          type: String
+      - name: 'destFqdns'
+        type: Array
+        immutable: true
+        description: |
+          Fully Qualified Domain Name (FQDN) which should be matched against traffic destination. Maximum number of destination fqdn allowed is 100.
+        item_type:
+          type: String
+      - name: 'srcRegionCodes'
+        type: Array
+        immutable: true
+        description: |
+          Region codes whose IP addresses will be used to match for source of traffic. Should be specified as 2 letter country code defined as per ISO 3166 alpha-2 country codes. ex."US" Maximum number of source region codes allowed is 5000.
+        item_type:
+          type: String
+      - name: 'destRegionCodes'
+        type: Array
+        immutable: true
+        description: |
+          Region codes whose IP addresses will be used to match for destination of traffic. Should be specified as 2 letter country code defined as per ISO 3166 alpha-2 country codes. ex."US" Maximum number of dest region codes allowed is 5000.
+        item_type:
+          type: String
+      - name: 'destThreatIntelligences'
+        type: Array
+        immutable: true
+        description: |
+          Names of Network Threat Intelligence lists. The IPs in these lists will be matched against traffic destination.
+        item_type:
+          type: String
+      - name: 'srcThreatIntelligences'
+        type: Array
+        immutable: true
+        description: |
+          Names of Network Threat Intelligence lists. The IPs in these lists will be matched against traffic source.
+        item_type:
+          type: String
+  - name: 'action'
+    type: String
+    description: 'The Action to perform when the client connection triggers the rule. Valid actions are "allow", "deny", "goto_next" and "apply_security_profile_group".'
+    required: true
+  - name: 'securityProfileGroup'
+    type: String
+    description: |
+      A fully-qualified URL of a SecurityProfile resource instance.
+      Example: https://networksecurity.googleapis.com/v1/projects/{project}/locations/{location}/securityProfileGroups/my-security-profile-group
+      Must be specified if action = 'apply_security_profile_group' and cannot be specified for other actions.
+  - name: 'tlsInspect'
+    type: Boolean
+    description: |
+      Boolean flag indicating if the traffic should be TLS decrypted.
+      Can be set only if action = 'apply_security_profile_group' and cannot be set for other actions.
+  - name: 'direction'
+    type: Enum
+    description: |
+      The direction in which this rule applies.
+    required: true
+    enum_values:
+      - 'INGRESS'
+      - 'EGRESS'
+  - name: 'enableLogging'
```

Comment:
This field does not properly update from `true` to `false`. Can we add `send_empty_value: true` here?